src/index.html

```diff
@@ -11,7 +11,202 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
     <script type="text/javascript" src="./main.js"></script>
+
+    <form 
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+    method="post"
+    >
+      <fieldset class="form-field">
+        <legend>Personal information</legend>
+        <div class="form_field">
+         Surname:
+         
+          <label>
+            <input 
+            type="text"
+            name="surname"
+            autocomplete="off"
+            >
+          </label>
+        </div>
+        
+        <div class="form_field">
+         Name:
+
+          <label>
+            <input 
+            type="text"
+            name="name"
+            autocomplete="off"
+            >
+          </label>
+        </div>
+        
+        <div class="form_field">
+          <label for="age"> How old are you</label>
+         
+          <label>
+            <input 
+            type="number"
+            id="age" 
+            name="age" 
+            min="1" 
+            max="100"
+            value="12"
+            >
+          </label>
+        </div>
+        
+        <div class="form_field">
+          <label> Full date of birth</label>
+
+          <label>
+            <input 
+            type="date"
+            name="dateOfbirth"
+            >
+          </label>
+        </div>
+
+        <div class="form_field">
+          I accept the term of the agreement
+         
+          <label>
+            <input 
+            type="checkbox"
+            name="agreement"
+            >
+          </label>
+        </div>
+      </fieldset>
+     
+      <fieldset class="form-field">
+        <legend>Registration</legend>
+
+        <div class="form_field">
+          E-mail:
+         
+          <label>
+            <input 
+            type="email"
+            name="email"  
+            placeholder="email@example.com"  
+            autocomplete="off"
+            >
+          </label>
+        </div>
+
+        <div class="form_field">
+          Password:
+         
+          <label>
+            <input 
+            type="password"
+            name="pass"
+            >
+          </label>
+        </div>
+      </fieldset>
+
+      <fieldset class="form-field">
+        <legend> An interesting fact about you!</legend>
+
+        <div class="form_field">
+          Do you love cats?
+
+          <label> 
+            <input 
+            type="radio"
+            name="loveCats" 
+            value="Yes"
+            > 
```

Comment:
```suggestion
            <input 
              type="radio"
              name="loveCats" 
              value="Yes"
            > 
```
Please fix code style everywhere.